src/main/java/org/llorllale/cactoos/matchers/HasContent.java

```diff
@@ -58,7 +59,11 @@ public HasContent(final String text) {
     public HasContent(final Text text) {
         this(
             new MatcherOf<>(
-                (String input) -> text.asString().equals(input), text
+                input -> text.asString().equals(input),
+                desc -> desc.appendText(
+                    new FormattedText("\"%s\"", text).asString()
+                ),
+                (actual, desc) -> desc.appendValue(actual)
```

Comment:
@baudoliver7 I think we need an `InputMatcher` similar to `TextMatcher`, could you add a todo? `InputMatcher` would be also in charge of caching the value of the `Input` so that it is read only once per invocation of the `Matcher`.

Reply:
@victornoel Of course !